02/PrintRecords.cpp

```diff
@@ -101,42 +95,39 @@ namespace samples
 				assert(false);
 				break;
 			}
-			PrintRecords(inputStream, recordCount);
+
+			string consoleRecord;
+
+			for (int i = 0; i < recordCount; ++i)
```

Comment:
shouldn't inputstream's position be set to beginning of stream before running this for loop?